src/index.js

```diff
@@ -1,13 +1,15 @@
 const express = require('express');
+const cors    = require ('cors');
 
 const { PopulateRouter, TimetableRouter, GCRouter } = require('./routes');
 
 const app = express();
 
-app.use(GCRouter);
-app.use(TimetableRouter);
-app.use(PopulateRouter);
-
-app.listen(3000, () => {
+app
+.use (cors ({})) // allow all
```

Comment:
Are you sure it will work? Have you tested it before creating pull request?

Reply:
yeah